packages/sitebuilder/src/constants/index.ts

```diff
@@ -6,8 +6,8 @@ export const ASSETS_URL = SITEBUILDER.assets_url;
 
 export const IMAGE_DIR = `${ ASSETS_URL }images/`;
 
-// export const CARDS: SetupCardInterface[] = SITEBUILDER.cards || [];
-export const CARDS: SetupCardInterface[] = cardsJson;
+export const CARDS: SetupCardInterface[] = SITEBUILDER.cards || [];
+//export const CARDS: SetupCardInterface[] = cardsJson;
```

Comment:
If this code is not intended to come back, remove the line completely instead of commenting it out.